workflows/maskrcnn-training/template.yaml

```diff
@@ -99,7 +81,7 @@ templates:
       workingDir: /mnt/src
     sidecars:
       - name: tensorboard
-        image: tensorflow/tensorflow:2.3.0
+        image: tensorflow/tensorflow:2.4.0
```

Comment:
Might as well use `onepanel/dl:0.17.0` here too since it will already be cached in this node.

Reply:
Sure.